docs/source/index.rst

```diff
@@ -15,26 +15,35 @@ geometries.
 GeoArrow
 --------
 
-cuSpatial proposes a new GeoArrow format from the fruit of discussions with the GeoPandas team. GeoArrow is a packed columnar data format for the six fundamental geometry types: Point, MultiPoint, Lines, MultiLines, Polygons, and MultiPolygons. MultiGeometry is a possibility that may be implemented in the future. GeoArrow uses packed coordinate and offset columns to define objects, which enables very-fast copy between CPU, GPU, and NIC.
+cuSpatial proposes a new GeoArrow format from the fruit of discussions
+with the GeoPandas team. GeoArrow is a packed columnar data format
```

Comment:
GeoArrow exists independently of cuSpatial. This makes it sound like we invented GeoArrow. I don't think this sentence belongs in our documentation. (CC @thomcom )

```suggestion
GeoArrow is a packed columnar data format
```

Reply:
That's fine. I had the first implementation, though GeoArrow diverged from my implementation a little as per the previous discussions. :D :D